app/assets/stylesheets/application.scss

```diff
@@ -34,17 +35,15 @@ $foot: #888888;
     font-style: italic;
     padding-bottom: 15px;
   }
-
-  .submit-form {
-    background: rgba(0, 0, 0, 0.6);
-    color: $white;
-    font: bold 12px/24px Helvetica, Sans-Serif;
-    margin-bottom: 40px;
-    margin-top: 10px;
-    padding: 1px;
-    padding-left: 10px;
-    padding-right: 10px;
-  }
+}
+.submit-form button {
+  font: bold 12px/24px Helvetica, Sans-Serif;
+  margin-bottom: 40px;
+  margin-top: 10px;
```

Comment:
Properties should be ordered font, margin-bottom, margin-left, margin-top, padding, padding-left, padding-right